### PR TITLE
fix(oauthdb): split creating oauthdb from main process startup

### PR DESCRIPTION
**Pull request by dannycoates**
possibly a fix for #6100

### CODE REVIEW

**Comment by dannycoates** [comment]
@chenba can you try this? I can't repro the bug locally very reliably

**Comment by chenba** [comment]
Ran it a few times, 100% success rate. 💯

File: packages/fxa-auth-server/scripts/test-local.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ node -r ts-node/register ./scripts/gen_keys.js
 node -r ts-node/register ./scripts/gen_vapid_keys.js
 node -r ts-node/register ./scripts/oauth_gen_keys.js
 node -r ts-node/register ../fxa-auth-db-mysql/bin/db_patcher > /dev/null
+node -r ts-node/register ./scripts/update-oauth-db.js
```
Comment (vbudhram) [inline packages/fxa-auth-server/scripts/test-local.sh:19]: nit: A name like `oauth-db-patcher` reads a little cleaner to me what this script would do.

**Review by vbudhram — COMMENTED** [review]
Also ran this a few times with 100% pass rate 🙌🏽

**Pull request merged**